README.md

```diff
@@ -1,430 +1,100 @@
 [![Build Status](https://travis-ci.com/stripe/stripe-cli.svg?token=eQWDVpt1sJR63TxbC1KA&branch=master)](https://travis-ci.com/stripe/stripe-cli)
+![GitHub](https://img.shields.io/github/license/stripe/stripe-cli)
 
-_🏗 The Stripe CLI is currently in beta! We're working on more features to make the experience great. If you have any [feedback](https://stri.pe/cli-feedback), find [issues](https://github.com/stripe/stripe-cli/issues), or would like to be involved in more active testing, please let us know!_
-
-The Stripe CLI is a command-line interface for Stripe that can:
-
-1. `login` to your Stripe account and authenticate the CLI
-2. `listen` for webhooks and forward them to a local server
-3. Run resource commands for things like `stripe charges create`
-4. Run `get`, `post`, and `delete` commands to the Stripe API
-5. `trigger` a limited set of webhook events
-6. Tail your test mode API request logs
-7. Pull Stripe status from status.stripe.com
-
-The main focus for this initial release is to improve the developer experience while integrating and testing webhooks. Interactions through the CLI are currently limited to test mode only.
-
-## Table of Contents
-
-* [Stripe CLI](#stripe-cli)
-  * [Installation](#installation)
-    * [macOS](#macos)
-    * [Linux](#linux)
-    * [Windows](#windows)
-    * [Docker](#docker)
-  * [Commands](#commands)
-    * [login](#login)
-    * [listen](#listen)
-    * [Resource commands](#resource-commands)
-    * [get, post, and delete](#get-post-and-delete)
-    * [trigger](#trigger)
-    * [logs tail](#logs-tail)
-    * [samples](#samples)
-    * [status](#status)
-    * [config](#config)
-    * [open](#open)
-  * [Developing the Stripe CLI](#developing-the-stripe-cli)
-    * [Installation](#installation-1)
-    * [Linting](#linting)
-    * [Tests](#tests)
-    * [Releasing](#releasing)
-  * [License](#license)  
-    
+The Stripe Command-line interface (CLI) makes it easy for you to build, test and manage your Stripe integration right from the terminal by integrating seamlessly with your Stripe account.
 
-## Installation
-
-### macOS
-
-_homebrew:_
-
-Run:
-
-```sh
-$ brew install stripe/stripe-cli/stripe
-```
-
-### Linux
-
-_Debian/Ubuntu-based distributions_:
-
-1. Add Bintray's GPG key to the apt sources keyring:
-
-```sh
-$ sudo apt-key adv --keyserver hkp://pool.sks-keyservers.net:80 --recv-keys 379CE192D401AB61
-```
-
-2. Add stripe-cli's apt repository to the apt sources list:
-
-```sh
-$ echo "deb https://dl.bintray.com/stripe/stripe-cli-deb stable main" | sudo tee -a /etc/apt/sources.list
-```
-
-3. Update the package list:
-
-```sh
-$ sudo apt-get update
-```
+**Feature highlights**:
 
-4. Install the CLI:
+- Securely test webhooks without relying on 3rd party software.
+- Trigger webhook events for easy testing
+- Tail your API request logs in real-time
+- Manage resources by interacting directly with the API to `create`, `retrieve`, `update` or `delete` any Stripe object
```

Comment:
`create`, `retrieve`, `update` or `delete` is a subset of what the commands do, should we try to explain more?

Reply:
We can just remove it and keep it concise to `Manage resources by interacting directly with the API`?

Reply:
Sounds good!